scripts/check_release.py

```diff
@@ -0,0 +1,53 @@
+import re
+import sys
+from pathlib import Path
+
+from git import Repo
```

Comment:
How do we get this dependenciy?

Reply:
(script-languages-cgrGsUDZ-py3.8) thomas@thomas-Standard-PC-Q35-ICH9-2009:~/Work/script-languages-release$ poetry show -t
exasol-script-languages-container-ci 0.4.0 Implements CI builds for script-language-container.
├── click >=8.0.3,<9.0.0
│   └── colorama * 
├── exasol-integration-test-docker-environment *
│   ├── click >=7.0 
│   │   └── colorama * 
│   ├── docker >=4.0.0 
│   │   ├── requests >=2.14.2,<2.18.0 || >2.18.0 
│   │   │   ├── certifi >=2017.4.17 
│   │   │   ├── charset-normalizer >=2,<3 
│   │   │   ├── idna >=2.5,<4 
│   │   │   └── urllib3 >=1.21.1,<1.27 
│   │   └── websocket-client >=0.32.0 
│   ├── gitpython >=2.1.0 
│   │   └── gitdb >=4.0.1,<5 
│   │       └── smmap >=3.0.1,<6 
│   ├── humanfriendly >=4.18 
│   │   └── pyreadline3 * 
│   ├── jinja2 >=2.10.1 
│   │   └── markupsafe >=2.0 
│   ├── jsonpickle >=1.1 
│   ├── luigi >=2.8.4 
│   │   ├── python-daemon * 
│   │   │   ├── docutils * 
│   │   │   └── lockfile >=0.10 
│   │   ├── python-dateutil >=2.7.5,<3 
│   │   │   └── six >=1.5 
│   │   ├── tenacity >=8,<9 
│   │   └── tornado >=5.0,<7 
│   ├── netaddr >=0.7.19 
│   ├── networkx >=2.3 
│   ├── pydot >=1.4.0 
│   │   └── pyparsing >=2.1.4 
│   ├── requests >=2.21.0 (circular dependency aborted here)
│   ├── simplejson >=3.16.0 
│   └── stopwatch.py >=1.0.0 
├── exasol-script-languages-container-tool *
│   ├── exasol-integration-test-docker-environment * 
│   │   ├── click >=7.0 
│   │   │   └── colorama * 
│   │   ├── docker >=4.0.0 
│   │   │   ├── requests >=2.14.2,<2.18.0 || >2.18.0 
│   │   │   │   ├── certifi >=2017.4.17 
│   │   │   │   ├── charset-normalizer >=2,<3 
│   │   │   │   ├── idna >=2.5,<4 
│   │   │   │   └── urllib3 >=1.21.1,<1.27 
│   │   │   └── websocket-client >=0.32.0 
│   │   ├── gitpython >=2.1.0 
│   │   │   └── gitdb >=4.0.1,<5 
│   │   │       └── smmap >=3.0.1,<6 
│   │   ├── humanfriendly >=4.18 
│   │   │   └── pyreadline3 * 
│   │   ├── jinja2 >=2.10.1 
│   │   │   └── markupsafe >=2.0 
│   │   ├── jsonpickle >=1.1 
│   │   ├── luigi >=2.8.4 
│   │   │   ├── python-daemon * 
│   │   │   │   ├── docutils * 
│   │   │   │   └── lockfile >=0.10 
│   │   │   ├── python-dateutil >=2.7.5,<3 
│   │   │   │   └── six >=1.5 
│   │   │   ├── tenacity >=8,<9 
│   │   │   └── tornado >=5.0,<7 
│   │   ├── netaddr >=0.7.19 
│   │   ├── networkx >=2.3 
│   │   ├── pydot >=1.4.0 
│   │   │   └── pyparsing >=2.1.4 
│   │   ├── requests >=2.21.0 (circular dependency aborted here)
│   │   ├── simplejson >=3.16.0 
│   │   └── stopwatch.py >=1.0.0 
│   ├── importlib-metadata >=4.6.0 
│   │   └── zipp >=0.5 
│   └── importlib-resources >=5.4.0 
│       └── zipp >=3.1.0 (circular dependency aborted here)
├── gitpython >=3.1.0
│   └── gitdb >=4.0.1,<5 
│       └── smmap >=3.0.1,<6 
└── pygithub >=1.55.0,<2.0.0
    ├── deprecated * 
    │   └── wrapt >=1.10,<2 
    ├── pyjwt >=2.0 
    ├── pynacl >=1.4.0 
    │   └── cffi >=1.4.1 
    │       └── pycparser * 
    └── requests >=2.14.0 
        ├── certifi >=2017.4.17 
        ├── charset-normalizer >=2,<3 
        ├── idna >=2.5,<4 
        └── urllib3 >=1.21.1,<1.27 
exasol-script-languages-container-ci-setup 0.5.0 Manages AWS cloud CI build infrastructure.
├── boto3 >=1.22.0,<2.0.0
│   ├── botocore >=1.27.31,<1.28.0 
│   │   ├── jmespath >=0.7.1,<2.0.0 
│   │   ├── python-dateutil >=2.1,<3.0.0 
│   │   │   └── six >=1.5 
│   │   └── urllib3 >=1.25.4,<1.27 
│   ├── jmespath >=0.7.1,<2.0.0 (circular dependency aborted here)
│   └── s3transfer >=0.6.0,<0.7.0 
│       └── botocore >=1.12.36,<2.0a.0 (circular dependency aborted here)
├── click >=8.1.3,<9.0.0
│   └── colorama * 
├── exasol-error-reporting-python rev 0.2.0
├── exasol-script-languages-container-ci *
│   ├── click >=8.0.3,<9.0.0 
│   │   └── colorama * 
│   ├── exasol-integration-test-docker-environment * 
│   │   ├── click >=7.0 (circular dependency aborted here)
│   │   ├── docker >=4.0.0 
│   │   │   ├── requests >=2.14.2,<2.18.0 || >2.18.0 
│   │   │   │   ├── certifi >=2017.4.17 
│   │   │   │   ├── charset-normalizer >=2,<3 
│   │   │   │   ├── idna >=2.5,<4 
│   │   │   │   └── urllib3 >=1.21.1,<1.27 
│   │   │   └── websocket-client >=0.32.0 
│   │   ├── gitpython >=2.1.0 
│   │   │   └── gitdb >=4.0.1,<5 
│   │   │       └── smmap >=3.0.1,<6 
│   │   ├── humanfriendly >=4.18 
│   │   │   └── pyreadline3 * 
│   │   ├── jinja2 >=2.10.1 
│   │   │   └── markupsafe >=2.0 
│   │   ├── jsonpickle >=1.1 
│   │   ├── luigi >=2.8.4 
│   │   │   ├── python-daemon * 
│   │   │   │   ├── docutils * 
│   │   │   │   └── lockfile >=0.10 
│   │   │   ├── python-dateutil >=2.7.5,<3 
│   │   │   │   └── six >=1.5 
│   │   │   ├── tenacity >=8,<9 
│   │   │   └── tornado >=5.0,<7 
│   │   ├── netaddr >=0.7.19 
│   │   ├── networkx >=2.3 
│   │   ├── pydot >=1.4.0 
│   │   │   └── pyparsing >=2.1.4 
│   │   ├── requests >=2.21.0 (circular dependency aborted here)
│   │   ├── simplejson >=3.16.0 
│   │   └── stopwatch.py >=1.0.0 
│   ├── exasol-script-languages-container-tool * 
│   │   ├── exasol-integration-test-docker-environment * (circular dependency aborted here)
│   │   ├── importlib-metadata >=4.6.0 
│   │   │   └── zipp >=0.5 
│   │   └── importlib-resources >=5.4.0 
│   │       └── zipp >=3.1.0 (circular dependency aborted here)
│   ├── gitpython >=3.1.0 (circular dependency aborted here)
│   └── pygithub >=1.55.0,<2.0.0 
│       ├── deprecated * 
│       │   └── wrapt >=1.10,<2 
│       ├── pyjwt >=2.0 
│       ├── pynacl >=1.4.0 
│       │   └── cffi >=1.4.1 
│       │       └── pycparser * 
│       └── requests >=2.14.0 (circular dependency aborted here)
├── exasol-script-languages-container-tool *
│   ├── exasol-integration-test-docker-environment * 
│   │   ├── click >=7.0 
│   │   │   └── colorama * 
│   │   ├── docker >=4.0.0 
│   │   │   ├── requests >=2.14.2,<2.18.0 || >2.18.0 
│   │   │   │   ├── certifi >=2017.4.17 
│   │   │   │   ├── charset-normalizer >=2,<3 
│   │   │   │   ├── idna >=2.5,<4 
│   │   │   │   └── urllib3 >=1.21.1,<1.27 
│   │   │   └── websocket-client >=0.32.0 
│   │   ├── gitpython >=2.1.0 
│   │   │   └── gitdb >=4.0.1,<5 
│   │   │       └── smmap >=3.0.1,<6 
│   │   ├── humanfriendly >=4.18 
│   │   │   └── pyreadline3 * 
│   │   ├── jinja2 >=2.10.1 
│   │   │   └── markupsafe >=2.0 
│   │   ├── jsonpickle >=1.1 
│   │   ├── luigi >=2.8.4 
│   │   │   ├── python-daemon * 
│   │   │   │   ├── docutils * 
│   │   │   │   └── lockfile >=0.10 
│   │   │   ├── python-dateutil >=2.7.5,<3 
│   │   │   │   └── six >=1.5 
│   │   │   ├── tenacity >=8,<9 
│   │   │   └── tornado >=5.0,<7 
│   │   ├── netaddr >=0.7.19 
│   │   ├── networkx >=2.3 
│   │   ├── pydot >=1.4.0 
│   │   │   └── pyparsing >=2.1.4 
│   │   ├── requests >=2.21.0 (circular dependency aborted here)
│   │   ├── simplejson >=3.16.0 
│   │   └── stopwatch.py >=1.0.0 
│   ├── importlib-metadata >=4.6.0 
│   │   └── zipp >=0.5 
│   └── importlib-resources >=5.4.0 
│       └── zipp >=3.1.0 (circular dependency aborted here)
├── jinja2 >=3.1.0
│   └── markupsafe >=2.0 
├── jsonschema >=3.2.0,<4.0.0
│   ├── attrs >=17.4.0 
│   ├── pyrsistent >=0.14.0 
│   └── six >=1.11.0 
└── pygithub >=1.55.0,<2.0.0
    ├── deprecated * 
    │   └── wrapt >=1.10,<2 
    ├── pyjwt >=2.0 
    ├── pynacl >=1.4.0 
    │   └── cffi >=1.4.1 
    │       └── pycparser * 
    └── requests >=2.14.0 
        ├── certifi >=2017.4.17 
        ├── charset-normalizer >=2,<3 
        ├── idna >=2.5,<4 
        └── urllib3 >=1.21.1,<1.27 
exasol-script-languages-container-tool 0.14.0 Script Languages Container Tool
├── exasol-integration-test-docker-environment *
│   ├── click >=7.0 
│   │   └── colorama * 
│   ├── docker >=4.0.0 
│   │   ├── requests >=2.14.2,<2.18.0 || >2.18.0 
│   │   │   ├── certifi >=2017.4.17 
│   │   │   ├── charset-normalizer >=2,<3 
│   │   │   ├── idna >=2.5,<4 
│   │   │   └── urllib3 >=1.21.1,<1.27 
│   │   └── websocket-client >=0.32.0 
│   ├── gitpython >=2.1.0 
│   │   └── gitdb >=4.0.1,<5 
│   │       └── smmap >=3.0.1,<6 
│   ├── humanfriendly >=4.18 
│   │   └── pyreadline3 * 
│   ├── jinja2 >=2.10.1 
│   │   └── markupsafe >=2.0 
│   ├── jsonpickle >=1.1 
│   ├── luigi >=2.8.4 
│   │   ├── python-daemon * 
│   │   │   ├── docutils * 
│   │   │   └── lockfile >=0.10 
│   │   ├── python-dateutil >=2.7.5,<3 
│   │   │   └── six >=1.5 
│   │   ├── tenacity >=8,<9 
│   │   └── tornado >=5.0,<7 
│   ├── netaddr >=0.7.19 
│   ├── networkx >=2.3 
│   ├── pydot >=1.4.0 
│   │   └── pyparsing >=2.1.4 
│   ├── requests >=2.21.0 (circular dependency aborted here)
│   ├── simplejson >=3.16.0 
│   └── stopwatch.py >=1.0.0 
├── importlib-metadata >=4.6.0
│   └── zipp >=0.5 
└── importlib-resources >=5.4.0
    └── zipp >=3.1.0

Reply:
maybe specify as direct dev dep

Reply:
Done